backend/src/main/tf/extensions.tf

```diff
@@ -0,0 +1,52 @@
+resource "aws_dynamodb_table" "users" {
```

Comment:
Not sure about general best practices, but `backend/src/main/tf/`isn't the first place I'd look for Terraform files.
I'd either pull it one level up under `backend/tf` or add a `tf` directory in the root of the project.

Reply:
agreed, moved to the module root